### ISSUE TITLE
"Failed to convert type" when re-exporting a wildcard import

### ISSUE THREAD

**Issue by christopherthielen** [TypeStrong/typedoc#1453]
I ran across this error when updating typedoc-plugin-external-module-name to 0.20.  My plugin's functional tests re-export a named splat import from a child module.

## Search terms

Failed to convert type when re-exporting a named wildcard import

## Actual Behavior

```
Warning: Failed to convert type: typeof import("/Users/cthielen/projects/downstream-cache/test/src/annotations/dir1/index") when converting reexport_dir1. Please report a bug.
```

## Steps to reproduce the bug

```
import * as dir1 from './dir1';
export const reexport_dir1 = dir1;
```

clone and test:
```
git clone git@github.com:christopherthielen/typedoc-plugin-external-module-name.git
cd typedoc-plugin-external-module-name
git checkout bugreport
yarn
npx typedoc src/index.ts
```

## Environment

-   Typedoc version: 0.20.11
-   TypeScript version: 4.1.3
-   Node.js version: 12.16.0
-   OS: big sur

**Comment by Gerrit0** [comment]
Fixed in 0.20.12, thanks for the report!

I wasn't entirely sure what to use as the names for these reference types... including user paths in docs isn't great. I opted for `__module`, unless it was a type in the form of `import("x").Y`, in which case I use `Y`.

**Comment by christopherthielen** [comment]
thank you @Gerrit0  for your attentiveness to this project
